### PR TITLE
hbase 2.4.14

### PR DESCRIPTION
**Pull request by cho-m**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

File: Formula/hbase.rb [hunk 1]
```diff
@@ -163,6 +162,9 @@ def post_install
     system "#{bin}/start-hbase.sh"
     sleep 10
     begin
+      system "ls", "-al", testpath
+      system "ls", "-al", testpath/"pid"
+      system "ps", "-ef"
       assert_match "Zookeeper", pipe_output("nc 127.0.0.1 #{port} 2>&1", "stats")
```
Comment (cho-m) [inline Formula/hbase.rb:168]: Test originally failed, but after adding debug commands it passed. May be a timing issue so can try increasing sleep on next run.

**Comment by cho-m** [comment]
Still having trouble getting LZO compression working with `opentsdb` #100699 on Monterey. I want to look into that issue before merge.

**Review by SMillerDev — DISMISSED** [review]

**Comment by cho-m** [comment]
Fixed ARM issue, which was due to JRuby version.

Still not sure about Monterey issue with LZO. I noticed that copying `libexec/lib/native/Mac_OS_X-x86_64-64/libgplcompression.0.dylib` from Big Sur bottle fixes the issue, so it looks like something is different about building that shared library on Monterey.

**Comment by cho-m** [comment]
Giving up on fixing LZO on Monterey. Can just merge update to 2.4.14.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**